### PR TITLE
Add support for rails 5

### PR DESCRIPTION
**Pull request by gmanley**

### CODE REVIEW

**Comment by jkthorne** [comment]
nice!

File: lib/vestal_versions/changes.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def version_changes
 			# creation. Incremental changes are reset when the record is saved because they represent
 			# a subset of the dirty attribute changes, which are reset upon save.
 			def incremental_version_changes
-				changes.slice(*versioned_columns)
+				saved_changes.slice(*versioned_columns)
```
Comment (gmanley) [inline lib/vestal_versions/changes.rb:51]: This is a key change to support rails 5.2. The behavior of changes changed in rails 5.2.

File: lib/vestal_versions/users.rb [hunk 2]
```diff
@@ -28,10 +28,14 @@ module UserVersionMethods
     included do
       belongs_to :user, :polymorphic => true
 
-      alias_method_chain :user, :name
-      alias_method_chain :user=, :name
+      alias_method :user_without_name, :user
+      alias_method :user, :user_with_name
```
Comment (gmanley) [inline lib/vestal_versions/users.rb:32]: Key change since alias_method_chain was removed

File: lib/vestal_versions/versions.rb [hunk 3]
```diff
@@ -31,7 +13,7 @@ def between(from, to)
       return [] if from_number.nil? || to_number.nil?
 
       condition = (from_number == to_number) ? to_number : Range.new(*[from_number, to_number].sort)
-      where(:number => condition).order("#{table_name}.#{connection.quote_column_name('number')} #{(from_number > to_number) ? 'DESC' : 'ASC'}").to_a
+      where(:number => condition).order(Arel.sql("#{table_name}.#{connection.quote_column_name('number')} #{(from_number > to_number) ? 'DESC' : 'ASC'}")).to_a
```
Comment (gmanley) [inline lib/vestal_versions/versions.rb:16]: Key change... to not get deprecation warnings in rails 5.2 and to support rails 6

**Comment by gmanley** [comment]
We can set up travis so it will run the tests against all the different rails/ruby versions supported. That is if we keep it public.

**Review by nimylian — APPROVED** [review]
LGTM

**Pull request merged**